ft/cachetable/cachetable.cc

```diff
@@ -3571,9 +3607,10 @@ ENSURE_POD(evictor);
 // This is the function that runs eviction on its own thread.
 //
 static void *eviction_thread(void *evictor_v) {
-    evictor* CAST_FROM_VOIDP(evictor, evictor_v);
+    evictor *CAST_FROM_VOIDP(evictor, evictor_v);
     evictor->run_eviction_thread();
-    return evictor_v;
+    toku_instr_delete_current_thread();
```

Comment:
toku_pthread_done() invokes toku_instr_delete_current_thread(), so the call here is odd. Fixed.